Questions.md

```diff
@@ -0,0 +1,40 @@
+# Q & A
+1. What about lack of experience on Git?
+
+> You can help them. Every one can and should learn such a popular version
+> control system. You might have noticed its ubiquity among the developers.
+> Have you ever seen ClearCase or Subversion tutorial on [Udacity][1]? Udacity
+> was founded by one of the World's most prolific [person][2] and I think it is
+> not an exaggeration when we say it is an online university that trains
+> "Silicon Valley".
+
+2. Which client application will be used (assuming not everybody will use
+bash)?
+
+> It is possible to use [SourceTree][3] or [GitHub Desktop][4], but I highly
```

Comment:
Notice the link references.

Reply:
Link references this way seem better than regular ``[name](URL)`` format. I will definitely update [my study notes on Markdown][1] format.

[1]: https://github.com/MehmetCagriK/Reflections2/blob/master/reflections/markupStudyNotes